### DISCUSSION TITLE
Adding List of Hackathon Submissions

### DISCUSSION THREAD

**Discussion by ajeetraina** [collabnix/docker-aiml-stories#14] [General]
https://github.com/collabnix/docker-aiml-stories?tab=readme-ov-file#aiml-hackathon

It would be great if we can add all the projects under Devpost into our Hackathon section

**Comment by ajeetraina** [comment]
https://docker.devpost.com/project-gallery

**Reply by bricefotzo** [reply]
Handled in #16
